enhanced/enhanced.go

```diff
@@ -166,19 +166,10 @@ func (e *Exporter) collectValues(ch chan<- prometheus.Metric, instance config.In
 		return fmt.Errorf("unable to get logs for instance %s: %s", instance.Instance, err)
 	}
 
-	wg := &sync.WaitGroup{}
-	defer wg.Wait()
-
-	wg.Add(len(values))
 	for key, value := range values {
-		go func(key string, value interface{}) {
-			defer wg.Done()
-
-			err := e.collectValue(ch, instance, key, value, l)
-			if err != nil {
-				log.Error(err)
-			}
-		}(key, value)
+		if err = e.collectValue(ch, instance, key, value, l); err != nil {
+			log.Error(err)
+		}
```

Comment:
This was requested in https://jira.percona.com/browse/PMM-1638 `Scrape all instance metrics in parallel.`

Reply:
It was about doing required API calls in parallel, not light CPU work.

Reply:
There were no api calls when scraping metrics for given instance.